eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -237,48 +237,49 @@ stages:
       - template: /eng/pipelines/templates/variables/globals.yml
     jobs:
       - ${{ each artifact in parameters.Artifacts }}:
-          - ${{if and(ne(artifact.skipPublishDevFeed, 'true'), or(eq(variables['SetDevVersion'], 'true'), and(eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal'))))}}:
```

Comment:
@scbedd do you know which variable was not working? 

Also @ckairen any idea why we are looking over the over the artifacts at this level as opposed to in one job?

Reply:
The issue was this evaluation `eq(variables['SetDevVersion'], 'true')` wasn't being honored. I tried a few different versions of this, but compile time this condition would never evaluate to true and include the individual integration release jobs.